.github/workflows/ci.yml

```diff
@@ -45,3 +45,15 @@ jobs:
 
       - name: Verify that canonical-data.json files adhere to canonical-schema.json
         run: yarn test
+      
+      - name: Verify that all UUIDs are indeed unique
```

Comment:
I added this check as a step to the schema validation job because, imo, it still validates the correctness of the JSON files, it's just that the schema spec doesn't know the idea of uniqueness.